server/models/requestModel.ts

```diff
@@ -49,6 +50,10 @@ const requestSchema = new Schema({
     type: Boolean,
     required: true,
     default: false
+  },
+  quantity: {
+    type: Number,
+    default: 1
```

Comment:
should this field be set to required? so no document is potentially made without it

Reply:
I didn't set to required because I thought if they didn't want to explicitly set the quantity to anything other than 1, then it didn't need to be required (and would take the default value of 1). Lmk what u think

Reply:
^ok this sounded very confusing lol. I didn't set required because if you want a quantity other than 1 then you will pass it , but by default (in most cases I would think) it would be 1.

Reply:
Updated.